src/main.ts

```diff
@@ -9,6 +11,18 @@ import * as string from './string';
 import Symbol from './Symbol';
 import WeakMap from './WeakMap';
 
+`!has('es6-promise')`;
+import './Promise';
+
+`!has('es6-symbol')`;
+import './Symbol';
+import has from '@dojo/has/has';
+
+if (has('host-browser')) {
```

Comment:
@kitsonk I couldn't just use the `!has()` string here because `pepjs` and `intersection-observer` error if loaded from node (they expect a browser). I can add the `has` checks before the `require` statements if you want?

Reply:
Let's talk about this, because while TypeScript allows it, we don't want to have a `require()` in the code...  We may need to use a dynamic `import('pepjs');` instead of the `require`.